client/my-sites/subscriptions/index.tsx

```diff
@@ -1,11 +1,48 @@
 import config from '@automattic/calypso-config';
 import SubscriptionManager from '@automattic/subscription-manager';
+import { useTranslate } from 'i18n-calypso';
 import page, { Callback } from 'page';
 import { createElement } from 'react';
 import { makeLayout, render } from 'calypso/controller';
 
+const SitesView: React.FunctionComponent = () => <span>Sites View</span>;
```

Comment:
Type definition is not required here, as the type can be inferred by TS.

Reply:
And in fact, `React.FunctionComponent` implies that the component allows to specify children, which is not true here. 😁